### PR TITLE
Updating to Version 0.2.0

### PR DESCRIPTION
**Pull request by emarinier**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,16 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.2.0] - 2024-06-24
```
Comment (apetkau) [inline CHANGELOG.md:6]: Can you update the date to tomorrow to better correspond to when we will make a release?

Also, can you add the following to the bottom of the readme file so the `[0.2.0]` links to the release tag:

```
[0.2.0]: https://github.com/phac-nml/gasclustering/releases/tag/0.2.0
```

**Review by apetkau — CHANGES_REQUESTED** [review]
Looks great. Thanks so much. Just one comment.

File: CHANGELOG.md (hunk 1 above)
Comment (emarinier) [reply CHANGELOG.md]: Updated in b7cfcdb1d0e323bca4ebc6b258b3e156a243da75

**Review by apetkau — APPROVED** [review]
Thanks so much. Approved 😄

**Pull request merged**